notebooks/helpers.py

```diff
@@ -61,12 +61,11 @@ def load_titanic(test_size=.25, feature_skip_tuple=(), random_state=1999):
         strings = {k: line_dict[k] for k in string_keys}
         numeric_labels[n] = line_dict["survived"]
 
-    sss = StratifiedShuffleSplit(numeric_labels, n_iter=1, test_size=test_size,
-                                 random_state=12)
+    sss = StratifiedShuffleSplit(n_iter=1, test_size=test_size, random_state=12)
     # This is a weird way to get the indices but it works
     train_idx = None
     test_idx = None
-    for train_idx, test_idx in sss:
+    for train_idx, test_idx in sss.split(numeric_data, numeric_labels):
```

Comment:
Hm, but that wouldn't work on sklearn 0.17 anymore, right? We would have to add sklearn 0.18 to the requirements and check_env.ipynb then (in bold font) since it could confuse people otherwise.  I think it would generally be a good idea to add travis tests for the notebooks to check if they all execute without error using the packages listed in the requirements. E.g., sth like `jupyter nbconvert --to notebook --execute mynotebook.ipynb`